packages/core/src/fonts/fakt/index.styl

```diff
@@ -0,0 +1,89 @@
+// Fakt
+
+// Blond - font-weight: 300
+
+@font-face {
+  font-family: 'Fakt-Web';
+  src: url("//cdn.auth0.com/styleguide/development/lib/font/fakt/FaktPro-Blond.eot?#iefix");
+  src: url("//cdn.auth0.com/styleguide/development/lib/font/fakt/FaktPro-Blond.eot?#iefix") format("eot"),
+       url("//cdn.auth0.com/styleguide/development/lib/font/fakt/FaktPro-Blond.woff") format("woff"),
+       url("//cdn.auth0.com/styleguide/development/lib/font/fakt/FaktPro-Blond.ttf") format("truetype")
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Fakt-Web';
+  src: url("//cdn.auth0.com/styleguide/development/lib/font/fakt/FaktPro-BlondItalic.eot?#iefix");
+  src: url("//cdn.auth0.com/styleguide/development/lib/font/fakt/FaktPro-BlondItalic.eot?#iefix") format("eot"),
+       url("//cdn.auth0.com/styleguide/development/lib/font/fakt/FaktPro-BlondItalic.woff") format("woff"),
+       url("//cdn.auth0.com/styleguide/development/lib/font/fakt/FaktPro-BlondItalic.ttf") format("truetype")
+  font-weight: 300;
+  font-style: italic;
+}
+
+// Normal - font-weight: 400
+
+@font-face {
+  font-family: 'Fakt-Web';
+  src: url("//cdn.auth0.com/styleguide/development/lib/font/fakt/FaktPro-Normal.eot?#iefix");
+  src: url("//cdn.auth0.com/styleguide/development/lib/font/fakt/FaktPro-Normal.eot?#iefix") format("eot"),
```

Comment:
this urls are wrong I think that the url will be something like `//cdn.auth0.com/styleguide/core/2.0.0/fonts/fakt/FaktPro-Normal.eot`